src/stores/annotations.ts

```diff
@@ -136,17 +136,20 @@ export const useAnnotationsStore = defineStore('annotations', () => {
 
     // When filtering by witness it can happen that a target is used for some other active annotation item,
     // In that case, we want to keep the level of highlighting it had and
-
+  
     filteredAnnotations.value
       .filter(annotation => !activeIds.includes(annotation.id))
       .forEach(annotation => {
         const selector = AnnotationUtils.generateTargetSelector(annotation);
         if (!selector) return;
         const selectorIsActive = activeIds.filter(id => selector === AnnotationUtils.generateTargetSelector(activeAnnotations.value[id])).length > 0;
-
-        if (!selectorIsActive && AnnotationUtils.getCurrentLevel(document.querySelector(selector)) < 0) {
```

Comment:
Why do you remove the level check here?

Reply:
Because there arises a bug with the target when it is null and it is not handled from the function getCurrentLevel(). Therefore I get first the target using document.querySelector() and then check if the target is null. If the target is null then no need to highlight the targets, therefore return;.